components/grid/toolbar.md

```diff
@@ -10,282 +10,175 @@ position: 45
 
 # Grid Toolbar
 
-The grid provides a toolbar where you can add a variety of actions that are not tied to a concrete row.
+The [Blazor Grid](https://demos.telerik.com/blazor-ui/grid/overview) toolbar can render built-in and custom tools. This article describes the built-in tools and shows how to add custom tools or customize the toolbar.
 
-To use a toolbar, define the `GridToolBarTemplate` tag of the grid. In it, you can use arbitrary HTML and components to get the desired layout, and also `GridCommandButton` instances in (you can read more about the features available in those buttons in the [Command Column](slug://components/grid/columns/command) article).
+## Built-in Tools
 
->note The toolbar is not associated with an item from the data source. The `Item` field on the click event handler argument of a `GridCommandButton` will always be `null` and the `Edit`, `Update`, `Cancel` commands do not work with it.
+By default, the [Blazor Grid](https://demos.telerik.com/blazor-ui/grid/overview) displays all its built-in tools in the order below. Use the tool tag if you need to define a tool explicitly in a [custom toolbar configuration](#toolbar-configuration).
```

Comment:
"By default, the Blazor Grid displays all its built-in tools in the order below."

This sentence means that as soon as you add the toolbar, all its tools are rendered by default. I guess this is not what happens, so the whole statement is misleading.